src/canister_sig_util/src/lib.rs

```diff
@@ -135,6 +136,133 @@ pub fn hash_bytes(value: impl AsRef<[u8]>) -> Hash {
     hasher.finalize().into()
 }
 
+pub fn delegation_signature_msg(
```

Comment:
Does this belong in the `canister_sig_utils` crate? I.e. shouldn't this crate provide the utilities around canister sigs that are independent of the signed message?

Not feeling super strongly, just wondering... Or is there a reason why _not_ to put it in `sig-verifier-js`?

Reply:
Also not feeling strongly -- I thought that it fits to the crate, as verifying delegations is one of the main usages of canister sigs, and building this message is somewhat involved.  Let's ponder a bit more, and maybe move it in a follow up PR (that will remove the `hash`-copy).